src/index.html

```diff
@@ -11,7 +11,137 @@
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form-field">Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form-field">How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label class="form-field">Full date of birth:
+          <input
+            type="date"
+            name="date"
+          >
+        </label>
+        <label>I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="term-of-agreement"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="container">
+        <legend>Registration</legend>
+        <label class="form-field">E-mail:
+          <input
+          type="email"
+          name="email"
+          placeholder="email@example.com"
+          required
+          >
+        </label>
+        <label>Password:
+          <input
+          type="password"
+          name="password"
+          minlength="6"
+          maxlength="16"
+          required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="container">
+        <legend>An interesting fact about you!</legend>
+        <label class="form-field">Do you love cats?
+          <label>
+            <input
+            type="radio"
+            name="love-cats"
+            value="Yes"
+            >
+            Yes
+          </label>
+          <label>
+            <input
+            type="radio"
+            name="love-cats"
+            value="No"
+            >
+            No
+          </label>
+        </label>
+        <label class="form-field">What is your favorite color?
+          <input
+          type="color"
+          name="color"
+          >
+        </label>
+        <label class="form-field">What time do you go to bed?
+          <input
+          type="time"
+          name="color"
+          step="1"
+          >
+        </label>
+        <label class="form-field">What are your favorite brand of cars?
+          <select multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+        <label>How do you rate our work?
+          <input
+          type="range"
+          name="rate"
+          value="1"
+          min="1"
+          max="10"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="container">
+        <legend>Additional info:</legend>
+        <label class="form-field">Comments:
+          <textarea
+            name="comments"
+            cols="20"
+            rows="2"
+          >
+          </textarea>
+        </label>
```

Comment:
```suggestion
          ></textarea>
        </label>
```


https://user-images.githubusercontent.com/102413088/189647922-8116d57b-7b32-44f9-893e-06c2dee9bd73.mov